README.md

```diff
@@ -1 +1,33 @@
-# angular-setup-readme
+# Angular Setup + directory structure
+
+## Overview
+
+We need to ensure that we have a level-headed way of structuring our applications from the beginning, otherwise we could end up with files everywhere, which means it's hard to know exactly what you are editing and where.
+
+## Objectives
+
+- Define the core structure of an Angular application
+- Create a feature based file structure
+
+## Core Structure
+
+Angular supports modules, and our application file structure will reflect this. Each module will have directives, templates, controllers and services.
```

Comment:
Try to clarify this a little bit more - what does a module represent? When learning Rails, students have a concept of a resource  ie, a blog would have posts, users, and comments. Is this what the modules have?